src/locales/fi.json

```diff
@@ -752,6 +753,24 @@
           "reject_offer": "Hylkää tarjous",
           "sports_services_prices": "Liikuntapalvelujen hinnasto",
           "sports_services_prices_url": "https://www.hel.fi/kulttuurin-ja-vapaa-ajan-toimiala/fi/palvelut/liikunta-ja-ulkoilu/maksut/"
+        },
+        "berth_invoice": {
+          "heading": "Venepaikan tiedot",
+          "heading_secondary": "Lasku venepaikasta. Jos haluat jatkaa sopimusta,\n maksa se ennen eräpäivää, {{date}}.",
+          "pay": "Siirry maksamaan",
+          "terminate_berth": "Irtisano venepaikka"
+        },
+        "contract": {
+          "label": "Sopimus",
+          "issued": "Luotu",
+          "edited": "Muokattu",
+          "singed": "Allekirjoitettu",
+          "status": "Tila",
+          "show": "Näytä sopimus"
+        },
+        "history": {
+          "label": "Venepaikkahistoria",
+          "berth": "paikka"
```

Comment:
Check that this is intended to be in lowercase

Reply:
Yeah, that's intentional